src/main/app/claims/claimStoreClient.ts

```diff
@@ -89,20 +91,21 @@ export class ClaimStoreClient {
             throw new ForbiddenError()
           }
         }
-        if (claim) {
-          return new Claim().deserialize(claim)
-        } else {
-          throw new Error('Call was successful, but received an empty claim instance')
-        }
+        return new Claim().deserialize(claim.body)
+      })
+      .catch(err => {
+        console.log(err)
```

Comment:
is this a WIP? if not this needs removing

Reply:
lint will block that dw ;), its wip but pretty much done, just been discussing with damian and rafal